search/signals.py

```diff
@@ -12,26 +12,32 @@
     Employment,
     Profile,
 )
+from dashboard.models import (
+    Certificate,
+    Enrollment,
+)
 from search.tasks import index_users, remove_user
 
 log = logging.getLogger(__name__)
 
 
 @receiver(post_save)
```

Comment:
I know this is old stuff, but unless I miss something I think this is wrong: you should attach the signal handler to the proper sender, otherwise this will be triggered for each `post_save` signal fired from any class.